# file: C:\Users\erena\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\scipy\integrate\_quadpack_py.py
# hypothesis_version: 6.140.2

[0.0, 5e-29, 1.49e-08, 'IntegrationWarning', 'Unknown error.', 'alg', 'alg-log', 'alg-loga', 'alg-logb', 'cauchy', 'cos', 'dblquad', 'depth', 'epsabs', 'epsrel', 'imag', 'neval', 'nquad', 'points', 'quad', 'real', 'sin', 'tplquad', 'unexpected kwargs', 'unknown']